new-log-viewer/src/services/decoders/JsonlDecoder.ts

```diff
@@ -16,6 +17,14 @@ import {
 } from "../../typings/logs";
 import LogbackFormatter from "../formatters/LogbackFormatter";
 
+/**
+ * Parsed JSONL.
+ * TODO: Add timestamp to interface by parsing in buildIdx and remove functionality from decode.
+ */
+interface JsonLogEvent {
+    level: LOG_LEVEL
+    jsonLog: JsonObject
+}
```

Comment:
Would it be better to add a LogEvent interface for all types of decoders, and here we extend the interface to include JSON logs specific properties? i.e. `level`s and `timestamp`s are generally expected for all types of logs, right?

Reply:
> The logEvents are private to the implementation so they don't necessarily need to be exposed to the interface.

That's a fair point. Let me trace the code and see what makes more sense.